tasks/build/Gaia/build.sh

```diff
@@ -23,11 +23,14 @@ for GOOS in $platforms; do
   make get_vendor_deps
   make install
 
-  if [[ "$GOOS" == "linux" ]]; then
+  if [ "$GOOS" = "linux" ]; then
     # Give the Linux binaries a prefix to make them like the others.
     mkdir --parents $TARGET/linux_amd64
     cp /go/bin/gaia* $TARGET/linux_amd64/
   fi
 done
 
-cp --recursive /go/bin/*_amd64 $TARGET/
+# copy build files to target dir
+if stat -t /go/bin/*_amd64 >/dev/null 2>&1; then
```

Comment:
I think I would prefer the command to error if nothing has been produced

Reply:
Should we exit before if we build only for linux?

Reply:
Ah probably I tested only a subset here, let's keep it simple, we will anyway change this again to remove `darwin` from our package.json